src/hooks/useCountries.ts

```diff
@@ -0,0 +1,25 @@
+import { useQuery } from "react-query";
+
+import { ApiClient } from "@/api/http-common";
+import { useEnvConfig } from "@/context/EnvConfig";
+
+export type TCountry = {
+  alpha2: string;
+  alpha3: string;
+  name: string;
+  official_name: string;
+  numeric: string;
+  flag: string;
+};
+
+export default function useCountries() {
+  const { CONCEPTS_API_URL } = useEnvConfig();
```

Comment:
Question: is this usable within this context as opposed to the SSR code of a given page?

Elsewhere we've used: `const { data } = await getEnvFromServer();` to get the env from the FE-server.

Reply:
We resolved this and realised we should retire the method i referred to as soon as we can in favour of your approach.